ecosystem/python/sdk/account_address.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
```

Comment:
nit: this is a little weird usage of shebang. Python3 specified here is used when you run `./account_address.py`. However, if my system's default python is 2.7 and I run `python account_address.py`, the python version is still 2.7. I don't think shebang is intended for such usage here.

Reply:
I don't know where I pulled it from, but this is here: https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take

I don't think the shebang does much unless if you run it with ./account_address.py -- one might argue that maybe it shouldn't even be there since there's no main. Something to chew on :D